website/src/components/HomePageFeatures/HomepageFeatures.module.css

```diff
@@ -38,6 +38,7 @@
 
 .featureDescription {
   font-size: medium;
+  text-align: left;
```

Comment:
I dont think it makes any difference?

Reply:
it makes changes in "Engage with Community" section, earlier the text was aligned center and now it is aligned to left.

Reply:
Revert back the changes.